src/hotspot/share/cds/metaspaceShared.cpp

```diff
@@ -567,7 +566,7 @@ void VM_PopulateDumpSharedSpace::doit() {
   // There may be pending VM operations. We have changed some global states
   // (such as vmClasses::_klasses) that may cause these VM operations
   // to fail. For safety, forget these operations and exit the VM directly.
-  os::_exit(0);
+  MetaspaceShared::exit_after_static_dump();
```

Comment:
The above comment can be removed since we have a similar comment above the definition of exit_after_static_dump()